### ISSUE TITLE
ssd_mobilenet on kitti

### ISSUE THREAD

**Issue by zeynali** [tensorflow/models#3330]
Hi.
i want to fine-tune kitti on ssd_mobilenet over 2 class , Pedestrian , Car .
first i successfully convert kitti to tfrecord , then i train up to 200k steps. but the result very bad. when i want to test , this show me 2 class but many many box in image of car and pedestrian . The NMS has not been respected , of course i don't know exactly Because of NMS ,  please hemp me . Has anyone ever done this?

**Comment by michaelisard** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!

**Comment by plieningerweb** [comment]
did you change anything besides the dataset and label_map?
there are some known bugs, e.g.  #2066 which lead to no learning.

**Comment by dextroza** [comment]
Hi there, I also cannot finetune ssd_mobilenet_v2_coco on KITTI, after 450k iterations the mAP(IOU=.50) is just .45. Should I change something more in pipeline.config except num_classes = 2? In KITTI labels I have two classes, car and pedestrian.

**Comment by zeynali** [comment]
@dextroza , when i fine-tuning ssd_mobilev2_coco on my own dataset , that is start to training but i get this warring : https://github.com/tensorflow/models/issues/3980
this error is happened for v1 but the same warring happed about v2 .
I'll say it in the old version of tensorflow repository that the folder mobilenet did't exist in `slim` the ssd_mobilev1_coco is ok and good trained

**Comment by dextroza** [comment]
@zeynali  If I understand well, you are able to train ssd_mobilenet on your own dataset with an old version of tensorflow/models. Could you tell me the commit of this old version and what version of ssd_mobilenet do you use?

**Comment by zeynali** [comment]
@dextroza , i mention that error is for ssd_mobilenetv1_coco but when i want to train ssd_mobilene v2 coco on i get same error but for MobileNetV2 , i get clone tensorflow github 1 months ago. my tensorflow version in 1.5 .
